core-sdk-samples/higgs-shop-sample-app/src/components/APIKeyHeaderBar/APIKeyHeaderBar.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button, Container, Typography } from '@mui/material';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import KeyIcon from '@mui/icons-material/Key';
+import React from 'react';
+
+const APIKeyHeaderBar: React.FC = () => {
+    return (
+        <Container
+            maxWidth={false}
+            sx={{
+                height: '50px',
+                backgroundColor: '#0F224C',
+                display: {
+                    xs: 'none',
+                    md: 'flex',
+                },
+                alignItems: 'center',
+                justifyContent: 'space-between',
+            }}
+        >
+            <Box
+                sx={{
+                    display: 'flex',
+                }}
+            >
+                <Typography variant='subtitle1' sx={{ px: '6px', py: '8px' }}>
+                    You are viewing the mParticle Web Sample App
+                </Typography>
+                <Button
+                    variant='text'
+                    startIcon={<GitHubIcon />}
+                    target='_blank'
+                    // TODO: Refactor url into constant once we integrate
```

Comment:
once we integrate what?

Reply:
Once I merge in the different UI Component PRs, I plan on doing a slight refactor so that reused urls can become constants. Might need to sync with product first to make sure that the urls are correct, so I left this TODO to remind me before we officially ship.